libsrc/telestrat/cputc.s

```diff
@@ -4,26 +4,50 @@
 ; void cputc (char c);
 ;
 
-        .export         _cputc, cputdirect
+        .export         _cputc, _cputcxy, cputdirect, display_conio
         .export         CHARCOLOR, OLD_CHARCOLOR, BGCOLOR, OLD_BGCOLOR
 
+        
+        .import         update_adscr
+        .import         popax
+
         .include        "telestrat.inc"
 
 cputdirect:
```

Comment:
The `cputdirect:` label must be moved to a line that is below the code that handles carriage returns and line feeds.

Reply:
@greg-king5  ok i have to look to this.

Reply:
@greg-king5  About cvline, it's fixed (commit below).